src/secex-genprotimgvm-scripts/genprotimg.bu

```diff
@@ -74,14 +74,16 @@ storage:
           ln -sf "../genprotimg.service" "${UNIT_DIR}/default.target.wants/"
 systemd:
   units:
-    - name: serial-getty@.service
-      dropins:
-        - name: autologin-core.conf
-          contents: |
-            [Service]
-            TTYVTDisallocate=no
-            ExecStart=
-            ExecStart=-/usr/sbin/agetty --autologin core  -o '-p -f core' --noclear %I $TERM
-      mask: false
-    - name: autovt@.service
-      mask: false
+    - name: autohalt.service
```

Comment:
I guess this is needed now because the `/sbin/halt` from `genprotimg.service` from before isn't executing?

Reply:
No, this poweroffs system after firstboot completed